crates/bevy_math/src/lib.rs

```diff
@@ -1,10 +1,29 @@
+#![warn(missing_docs)]
+
+//! Provides math types and functionalities for the bevy game engine.
+//!
+//! The commonly used types are vectors, rectangles and sizes.
+//!
+//! # Example
+//!
+//! ```rust
+//! use bevy_math::prelude::*;
+//! ```
+
```

Comment:
I don't think this code example is really useful, and we don't want to encourage using `bevy_math` standalone

Reply:
Understandable. I wasn't sure if the bevy crates should be documented as standalone or just assume that everyone is using bevy as a whole (which probably is the more likely case). I had a version where I had the following code which didn't work, because bevy is not a dependency of bevy_math. Adding the ignore keyword would make it harder to keep this up to date if we change something about the path in the future.

```rust
use bevy::math::prelude::*;
```

Reply:
Yeah, there's only a few crates that are worth using standalone: namely `bevy_ecs` and `bevy_reflect`. Everything else is a bit nonsensical outside of the Bevy ecosystem.

Reply:
Makes sense. I'll remove these examples then as stated below.